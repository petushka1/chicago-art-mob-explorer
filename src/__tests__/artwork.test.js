/* eslint-disable */
import { screen, render } from '@testing-library/react';
import Artwork from '../components/artwork';

test('renders Artwork', () => {
  render(
      <Artwork artwork={{
         image_id: 1,
         title: 'Unknown',
         date_start: 'Unknown',
         date_end: 1800,
      }}
      />
  );
  expect(screen.getByRole('heading', { level: 3 })).toMatchSnapshot();
});