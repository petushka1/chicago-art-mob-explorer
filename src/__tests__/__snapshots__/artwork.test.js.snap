// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Artwork 1`] = `
<h3>
  Unknown
</h3>
`;

exports[`renders Artwork 2`] = `
<h3>
  Unknown
</h3>
`;
